GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.cs

```diff
@@ -16,6 +16,7 @@ public partial class AppearanceSettingsPage : SettingsPageWithHeader
         private readonly TranslationString _noDictFile = new TranslationString("None");
         private readonly TranslationString _noDictFilesFound = new TranslationString("No dictionary files found in: {0}");
         private readonly TranslationString _noImageServiceTooltip = new TranslationString("A default image, if an email address has no matching Gravatar image.\r\nSee http://en.gravatar.com/site/implement/images#default-image for more details.");
+        private readonly TranslationString _authorDateSortWarningTooltip = new TranslationString("Sorting by author date may negatively impact performance.");
```

Comment:
Suggestions:
Sorting by author date may delay displaying the revision graph

General performance is not affected